### DISCUSSION TITLE
Changing to latest version on panda and ark-ui  is preventing CSS  in our Storybook from compiling.

### DISCUSSION THREAD

**Discussion by willTastyMakers** [chakra-ui/panda#2217] [Q&A]
Hi Community,
We have created a component library using panda-css and ark-ui. After updating to the latest,
``` 

    "@ark-ui/anatomy": "^2.2.0",
    "@ark-ui/react": "^2.1.1",
      ------
    "@pandacss/dev": "^0.31.0",
    "@pandacss/preset-panda": "^0.31.0",
    "@pandacss/types": "^0.31.0",
 
      --------
```
we are finding that our component stories are not effectively getting stylized with css. Basically, when we click on the story to review the component in storybook it appears bereft of any our style declarations from styled-system [css, recipes, jsx ( Box, Stack, etc.]  Interestingly, all it takes to get the css to compile is to force an HMR to occur by adding a space to the panda.config.ts file and saving it. Once that occurs all of the css appears.

Has anyone else experienced this behavior, and if so and your were able to fix it, could please expound on how to go about it?

Thank you.

**Comment by astahmer** [comment]
if you're using JSX style props and havent set any `jsxFramework` in your config, [this might be why](https://github.com/chakra-ui/panda/issues/761)

otherwise, can you make a minimal reproduction ?
